src/test.ts

```diff
@@ -317,26 +317,26 @@ describe('Trailing spaces', () => {
   it('Tab-Space-Linebreak removed', () => {
     const before = dedent`
         # H1
-        line with trailing tab and spaces    
```

Comment:
all the tests and examples for the trailing spaces rule got messed up, can you revert those changes so they test removing trailing spaces?

Reply:
Ah, apologies. My editor automatically strips trailing whitespace and I didn't consider that the tests with trailing whitespace would obviously be affected. Good catch!

Fixed in new commit